ArduPlane/Plane.h

```diff
@@ -996,6 +999,7 @@ class Plane : public AP_Vehicle {
     void read_airspeed(void);
     void rpm_update(void);
     void accel_cal_update(void);
+    void read_aoa(void);                         //This line added by Cole
```

Comment:
Let's lose the `This line added by Cole` throughout this patchset.  You probably know enough `git` now to remove the crutch